model/src/main/proto/format_import_proto_library.bzl

```diff
@@ -1,36 +1,37 @@
-def format_import_proto_library(name, src, deps):
-  '''
-  This macro exists as a way to build proto files that contain import statements in both Gradle
-  and Bazel.
-  This macro formats the src file's import statements to contain a full path to the file in order
-  for Bazel to properly locate file.
+"""
+This macro exists as a way to build proto files that contain import statements in both Gradle
```

Comment:
Similarly, this is the documentation for the macro. Ditto for all other macro files in this PR.

Reply:
Done